importer.py

```diff
@@ -226,7 +269,7 @@ def print_not_imported(self):
     spotify_oauth.add_argument('--id', required=True, help='Client ID of your Spotify app')
     spotify_oauth.add_argument('--secret', required=True, help='Client Secret of your Spotify app')
 
-    parser.add_argument('-t', '--token', required=True, help='Token from music.yandex.com account')
```

Comment:
why not required anymore?

Reply:
cause this way one could import from JSON only

Reply:
understood, but why do we create yandex_music client when there is no token in args? did i miss smth?

Reply:
c62e34c

Reply:
9ba9417